engines/order_management/spec/services/order_management/subscriptions/summary_spec.rb

```diff
@@ -56,6 +56,15 @@ module Subscriptions
         end
       end
 
+      describe "record_subscription_issue" do
+        let(:subscription) { double(:subscription, id: 101) }
```

Comment:
we should rely on an RSpec's verifying double such as https://relishapp.com/rspec/rspec-mocks/docs/verifying-doubles/using-an-instance-double to avoid the double drifting away from Susbcription's interface.

Reply:
Is it worth it if the only method being used is `#id`?